### ISSUE TITLE
Slideshows Don't Work in Multisite Installs

### ISSUE THREAD

**Issue by karks88** [WebDevStudios/StartBox#38]
Seems like there is a bug with the slideshow in Mutlisite installs.  The first site I install in the network, the slideshows work great.  Every site after that in the network refuses to display the slideshow when I use the "<?php echo do_shortcode('[slideshow id="xx"]'); ?>" code.

The source code of the slideshow appears in the page, but it does not display.  I've had this happen to at least 2 Multisite installs on different servers.

**Comment by brichards** [comment]
Interesting. http://demo.wpstartbox.com is a multisite install, as is http://playground.wpstartbox.com, and both of them render slideshows perfectly well across all sites.

Are you attempting to display a slideshow from 1 site across all the other sites? If so, that's definitely not supported.

What versions of WP and SB are you running on this network?

**Comment by karks88** [comment]
Hi Brian - I'm using WP 3.51 and StartBox 2.6.  I did create a new slideshow for each site in the network.  It's funny, if I go into the View Slideshow link in the admin, it shows up and works.  If I place the shortcode into the page template, there's nothing.

**Comment by karks88** [comment]
Perhaps this might help.  It's the source code for an example slider I tried to use on the 2nd site in my network:

div class="slider_wrapper" style="width:0px"
img width="" height="" title="" alt="" src="http://xxx.xxx.com/fia/usa/wp-content/themes/startbox/includes/scripts/timthumb.php?src=http://xxx.xxx.com/fia/usa/wp-content/themes/startbox/images/nophoto.jpg&w=&h=&a=tc&zc=1&q=100"
span class="slide_caption"/span
/div
(code changed to display in in forum)

Also of note, the images inside the slideshow area of the admin do not show up inside subsequent network sites.  So, for example, if I load in image it just doesn't appear, everything is blank.

Apparently it's just not finding the images.

**Comment by brichards** [comment]
AH! It's a timthumb issue (and possibly combined with the slideshow having no specific width or height). To that, I have good news: timthumb has been eradicated in the forthcoming 2.7 release, and so that should eliminate the problem here.

You can try 2.7 early, if you like, by downloading it here: https://github.com/WebDevStudios/StartBox/tree/2.7 (note: make sure you rename the unzipped theme directory to /startbox/, otherwise it'll break).

In fact, if you'd be willing to help test it and confirm nothing breaks in your development site that would help provide some fantastic feedback! Create new issues for any new bugs it might uncover (hopefully there will be none).

**Comment by karks88** [comment]
Thanks Brian!  I'll be glad to test it out and I'll confirm that it works here.

**Comment by karks88** [comment]
Well, sadly enough, I just did a completely fresh install of WP Multisite using 2.7 (new database as well) and unfortunately, now the images do not show up in any of the slideshows I try to create (in the admin or the live site).  Wish I had better news!  I tried clearing browser cache in case that had anything to do with it but no luck.

**Comment by brichards** [comment]
Alright, we'll do some testing and see what we can come up with. Standby!

**Comment by karks88** [comment]
Thanks much!

**Comment by karks88** [comment]
My apologies if this is not the right area to post this, but since it is related (and for some reason I can't login at WPStartBox.com to use the community forum), is there a way to disable the Slideshows?  I am using a different plugin using Nivo Slider and I think it may be causing some interference.

**Comment by brichards** [comment]
Yes it is! Just use this code in your child theme and they'll be gone:

``` php
// Disable StartBox's native slideshow feature
disable_startbox_slideshows() {
    remove_theme_support( 'sb-slideshows' );
}
add_action( 'after_setup_theme', 'disable_startbox_slideshows' );
```

**Comment by karks88** [comment]
Wonderful - thanks a million for all you guys do!
